fgpyo/collections/__init__.py

```diff
@@ -49,13 +49,12 @@
 
 `PeekableIterator`'s constructor supports creation from iterable objects as well as iterators.
 
-Module Contents
-~~~~~~~~~~~~~~~
+## Module Contents
 
 The module contains the following public classes:
 
-    - :class:`~fgpyo.collections.PeekableIterator` -- Iterator that allows you to peek at the
-        next value before calling next
+- [`PeekableIterator`](index.md#fgpyo.collections.PeekableIterator) -- Iterator that allows you to
+  peek at the next value before calling next.
```

Comment:
I think we can kill these sections now because of the new layout with the index.  And then it's one less thing to forget to update.